### ISSUE TITLE
[Deque Analysis] DSpace Green buttons/badges have insufficient color contrast for white text

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1163]
## Deque Analysis Summary

Buttons/badges which use a DSpace-like green background have color contrast issues.  This appears in several locations:

1. MyDSpace page, "archived" status green badges: Element has insufficient color contrast of 1.69 (foreground color: `#e8ebf3`, background color: `#92c642`) Contrast must be 4.5 or more."
    * Deque issue ticket IDs: 470124
2. Edit Item page, Metadata tab, "Add" button (both normal and hover over)
    * Normal button: Element has insufficient color contrast of 2.02 (foreground color: `#ffffff`, background color: `#92c642`)
    * Hover over button: Button text color: `#FFFFFF`, Background color: `#7AA832`, Contrast ratio: 2.8:1
    * Deque issue ticket IDs: 470728, 471447
3. Edit Item page, Bitstreams tab, "Upload" button (both normal & hover over)
    * Normal button: Element has insufficient color contrast of 2.02 (foreground color: `#ffffff`, background color: `#92c642`)
    * Hover over button: Button text color: `#FFFFFF`, Background color: `#7EAD34`, Contrast ratio: 2.7:1
    * Deque issue ticket IDs: 470733, 471056

## Recommended Fix

Two options.  Either make the green color darker.  Or, just change the text color to be *dark* instead of light (e.g. `#363636` instead of `#FFFFFF`).  I selected that color based on https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

I think I'd rather switch the text color (and keep the DSpace green) to dark grey/black.  However, I'd leave it up to the develop to change to green if it looks better.

While we are at it, we may want to match up the green color of the "hover over" for (2) and (3).  Changing both to a background of `#7EAD34` seems reasonable (as that color works better with `#363636` if we choose that for the text)

## More Information / Tools

* As necessary, see mentioned Issue IDs in our assessment results (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues
* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work
* Deque's color contrast tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

**Comment by michdyk** [comment]
@tdonohue Could you assign me to this issue? My team and I have investigated the accessibility of the system recently, so I guess it will be great to have done this task until we are fresh after our initial analysis.

**Comment by michdyk** [comment]
@tdonohue please assign my colleague @Dawnkai to this task. He is going to provide the PR for this issue. Also when it will be accepted, I kindly ask to give credits for this bug fix contribution to PCG Academia. Thanks
